concrete_datastore/api/v1_1/serializers.py

```diff
@@ -29,6 +29,12 @@
 concrete = apps.get_app_config('concrete')
 
 
+class ProcessRegisterSerializer(serializers.Serializer):
+    application = serializers.CharField(max_length=200, required=True)
+    instance = serializers.CharField(max_length=200, required=True)
+    token = serializers.CharField(max_length=40, required=True)
```

Comment:
Why such a limitation ? If an uuid v4 is not required (Char here), a 255 char long should be possible.

Reply:
this token is used to generate un instance of `AuthToken` with `key=token`. AuthToken enherits from `from rest_framework.authtoken.models.Token` (the default `key` field has a max_length of 40 chars). So if we increase `max_length` of the serialiser's field `token` we shuld also increase it in field `key` of `AuthToken`. What do you think ?

Reply:
In that case, the serializer field should be an instance of `serializers.UUIDField`